src/neo4j_genai/retrievers/hybrid.py

```diff
@@ -157,5 +164,11 @@ def search(
         search_query = get_search_query(
             SearchType.HYBRID, retrieval_query=self.retrieval_query
         )
+
+        logger.debug(
+            f"HybridCypherRetriever Cypher parameters: {json.dumps(parameters)}"
```

Comment:
Do we have to dump the parameters here? I'm asking because it may contain non JSON serializable values, like UUIDs (this particular example would fail at the query level later on, but it makes more sense to me than failing because of a log)

Reply:
Interesting point. When debugging, you sure want to have the information needed to debug the issue.
How would you do this?

Reply:
I would pass the parameters dict to the logger, without formatting. The logger will take care of "stringifying" each item.

Reply:
Thanks, updated